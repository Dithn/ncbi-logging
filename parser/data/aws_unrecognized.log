7dd4dcfe9b004fb7433c61af3e87972f2e9477fa7f0760a02827f771b41b3455 sra-pub-src-1 [26/May/2020:13:32:00 +0000] 10.11.149.94 arn:aws:iam::959185930736:user/david.yadin@kymab.com 186D7BF85015E3CE REST.HEAD.BUCKET - "HEAD / HTTP/1.1" 403 AccessDenied 243 - 196 - "-" "aws-sdk-java/1.11.160 Linux/4.4.0-1107-aws OpenJDK_64-Bit_Server_VM/11.0.7+10/11.0.7" - uYoz2qqYhiKUJCM3jMzDLO7the4EIjFZp6G9VLtn/jY1crCHDKHf5G3r1Cb9rmqsuOwcRrNv28k= SigV4 ECDHE-RSA-AES128-GCM-SHA256 AuthHeader sra-pub-src-1.s3.amazonaws.com TLSv1.2

7dd4dcfe9b004fb7433c61af3e87972f2e9477fa7f0760a02827f771b41b3455 sra-pub-src-1 [26/May/2020:16:00:16 +0000] 3.235.179.196 arn:aws:sts::784757538848:assumed-role/TulipRWAccess/i-00a5d1f5e0d3cdc67 1B6E92A991321AE8 REST.HEAD.OBJECT SRR9154112/*.fastq* "HEAD /SRR9154112/%2A.fastq%2A HTTP/1.1" 404 NoSuchKey 290 - 43 - "-" "aws-cli/1.16.300 Python/2.7.16 Linux/4.14.177-139.254.amzn2.x86_64 botocore/1.13.36" - 9cLyZqXg5N2mXetfQMtvMhhbczeIiHy+9UxgBSDV6UkpNCZxnUN7cNN/4TDc2oBlMk1DAUtG0GA= SigV4 ECDHE-RSA-AES128-GCM-SHA256 AuthHeader sra-pub-src-1.s3.amazonaws.com TLSv1.2

7dd4dcfe9b004fb7433c61af3e87972f2e9477fa7f0760a02827f771b41b3455 sra-pub-src-1 [26/May/2020:18:42:08 +0000] 130.14.28.7 arn:aws:iam::783971887864:user/ignatovi 9713E7798F9A44BA REST.COPY.OBJECT_GET SRR9211507/100100.mapped.1.fq.1 - - - - 351400 - - - - - QS2B/AbVsYy0J+Ce10atUPXDMX6vu6UOwEYF+EEovqNazgm8aO1xGpNN29+hN0KjCYv0VOTvsdI= SigV4 ECDHE-RSA-AES128-GCM-SHA256 AuthHeader sra-pub-sars-cov2.s3.amazonaws.com TLSv1.2

7dd4dcfe9b004fb7433c61af3e87972f2e9477fa7f0760a02827f771b41b3455 sra-pub-src-1 [25/May/2020:22:32:08 +0000] 52.54.203.43 arn:aws:sts::783971887864:assumed-role/sra-developer-instance-profile-role/i-04b64e15519efb678 B20C3FBA17A4BC38 REST.GET.OBJECT SRR9158220/A118_PF_4_1.fq.1 "GET /SRR9158220/A118_PF_4_1.fq.1 HTTP/1.1" 206 - 8388608 10679416372 230 182 "-" "aws-cli/1.16.249 Python/2.7.16 Linux/4.14.138-89.102.amzn1.x86_64 botocore/1.12.239" - +IfPMMJukaeGXc7cHIdkcLUDPcCCQVPtmWh04I9UUahItPSQxFgDgNUQQbh/ysxI4n4gHf5NK7M= SigV4 ECDHE-RSA-AES128-GCM-SHA256 AuthHeader sra-pub-src-1.s3.amazonaws.com TLSv1.2
922194806485875312b252374a3644f1feecd16802a50d4729885c1d11e1fd37 sra-pub-run-1 [22/Mar/2019:18:05:28 +0000] 3.89.6.210 arn:aws:iam::783971887864:user/ignatovi 31D289F437B52130 REST.HEAD.OBJECT . "HEAD /. HTTP/1.1" 404 NoSuchKey 272 - 23 - "-" "aws-cli/1.16.129 Python/3.6.7 Linux/4.15.0-1032-aws botocore/1.12.119" - dPgBpCGq9rQ7ZciElvjyXYPzKZV/WO1Kzthf39aEE7/VJiuOve05tKd5zgaCJlWYXnS89nw+YFo= SigV4 ECDHE-RSA-AES128-GCM-SHA256 AuthHeader sra-pub-run-1.s3.amazonaws.com TLSv1.2
922194806485875312b252374a3644f1feecd16802a50d4729885c1d11e1fd37 sra-pub-run-1 [22/Mar/2019:18:05:28 +0000] 3.89.6.210 arn:aws:iam::783971887864:user/ignatovi 31D289F437B52130 REST.HEAD.OBJECT . "HEAD /. HTTP/1.1" 404 NoSuchKey 272 - 23 - "-" "aws-cli/1.16.129 Python/3.6.7 Linux/4.15.0-1032-aws botocore/1.12.119" - dPgBpCGq9rQ7ZciElvjyXYPzKZV/WO1Kzthf39aEE7/VJiuOve05tKd5zgaCJlWYXnS89nw+YFo= SigV4 ECDHE-RSA-AES128-GCM-SHA256 AuthHeader sra-pub-run-1.s3.amazonaws.com TLSv1.2