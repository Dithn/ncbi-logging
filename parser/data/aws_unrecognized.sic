7dd4dcfe9b004fb7433c61af3e87972f2e9477fa7f0760a02827f771b41b3455 sra-pub-src-1 [25/May/2020:22:54:57 +0000] 52.54.203.43 arn:aws:sts::783971887864:assumed-role/sra-developer-instance-profile-role/i-04b64e15519efb678 EF87C0499CB7FDCA REST.HEAD.OBJECT ERR792423/m150101_223627_42225_c100719502550000001823155305141526_s1_p0.bas.h5.1 \"HEAD /ERR792423/m150101_223627_42225_c100719502550000001823155305141526_s1_p0.bas.h5.1 HTTP/1.1\" 200 - - 1318480 30 - \"-\" \"aws-cli/1.16.249 Python/2.7.16 Linux/4.14.138-89.102.amzn1.x86_64 botocore/1.12.239\" - Is1QS5IgOb006L7yAqIz7zKBtUIxbAZgzvM1aQbbSHXeKUDoSVfPXro2v1AN4gf7Ek5VTV2FeV8= SigV4 ECDHE-RSA-AES128-GCM-SHA256 AuthHeader sra-pub-src-1.s3.amazonaws.com TLSv1.2
7dd4dcfe9b004fb7433c61af3e87972f2e9477fa7f0760a02827f771b41b3455 sra-pub-src-2 [25/May/2020:22:17:41 +0000] 52.54.203.43 arn:aws:sts::783971887864:assumed-role/sra-developer-instance-profile-role/i-04b64e15519efb678 A3EE3F7C5D6EFE4F REST.GET.OBJECT_TAGGING SRR9670674/ALAL1_siRNA_rep1.fastq.gz.1 \"GET /SRR9670674/ALAL1_siRNA_rep1.fastq.gz.1?tagging HTTP/1.1\" 200 - 195 - 41 39 \"-\" \"aws-cli/1.16.249 Python/2.7.16 Linux/4.14.138-89.102.amzn1.x86_64 botocore/1.12.239\" - 7VRSSuE9gOSrWS5y0YUSto9bdATKDvU2nIuiBtEoMtoSHzZMOs+adDWjpy9dU6pkDkytmWqinFU= SigV4 ECDHE-RSA-AES128-GCM-SHA256 AuthHeader sra-pub-src-2.s3.amazonaws.com TLSv1.2
7dd4dcfe9b004fb7433c61af3e87972f2e9477fa7f0760a02827f771b41b3455 sra-pub-src-1 [25/May/2020:22:31:20 +0000] 52.54.203.43 arn:aws:sts::783971887864:assumed-role/sra-developer-instance-profile-role/i-04b64e15519efb678 1A45BC041C04BA85 REST.HEAD.OBJECT SRR9158220/A118_PF_4_1.fq.1 \"HEAD /SRR9158220/A118_PF_4_1.fq.1 HTTP/1.1\" 200 - - 10679416372 93 - \"-\" \"aws-cli/1.16.249 Python/2.7.16 Linux/4.14.138-89.102.amzn1.x86_64 botocore/1.12.239\" - bwwtFxVev8hXqapPwdVpejaskbJ8rOgY91qNOI6SdGqrxlvKvPzkr7NU/X7azJmtLI8wQSGxvO4= SigV4 ECDHE-RSA-AES128-GCM-SHA256 AuthHeader sra-pub-src-1.s3.amazonaws.com TLSv1.2
7dd4dcfe9b004fb7433c61af3e87972f2e9477fa7f0760a02827f771b41b3455 sra-pub-src-1 [25/May/2020:22:31:20 +0000] 52.54.203.43 arn:aws:sts::783971887864:assumed-role/sra-developer-instance-profile-role/i-04b64e15519efb678 D39CE53FE30FD35C REST.GET.OBJECT SRR9158220/A118_PF_4_1.fq.1 \"GET /SRR9158220/A118_PF_4_1.fq.1 HTTP/1.1\" 206 - 8388608 10679416372 311 272 \"-\" \"aws-cli/1.16.249 Python/2.7.16 Linux/4.14.138-89.102.amzn1.x86_64 botocore/1.12.239\" - jCL/sBSnUyAlhbRD7zCp7RqLm66zzh+zY924BgY26d9p9juoLHSifcuFqIv4aWj5aLPokM6DYVg= SigV4 ECDHE-RSA-AES128-GCM-SHA256 AuthHeader sra-pub-src-1.s3.amazonaws.com TLSv1.2
7dd4dcfe9b004fb7433c61af3e87972f2e9477fa7f0760a02827f771b41b3455 sra-pub-src-1 [25/May/2020:22:31:21 +0000] 52.54.203.43 arn:aws:sts::783971887864:assumed-role/sra-developer-instance-profile-role/i-04b64e15519efb678 44B966901DC2707B REST.GET.OBJECT SRR9158220/A118_PF_4_1.fq.1 \"GET /SRR9158220/A118_PF_4_1.fq.1 HTTP/1.1\" 206 - 8388608 10679416372 180 102 \"-\" \"aws-cli/1.16.249 Python/2.7.16 Linux/4.14.138-89.102.amzn1.x86_64 botocore/1.12.239\" - l+47+kEn5M6u1GttJzZF2UwRYN+mnu6G+nK1lSOowMfmYbPK4PSsCBqQQUK7QBEBnxWPC4do0D8= SigV4 ECDHE-RSA-AES128-GCM-SHA256 AuthHeader sra-pub-src-1.s3.amazonaws.com TLSv1.2

