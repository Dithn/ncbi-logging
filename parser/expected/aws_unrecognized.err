converting AWS format
total : 9
accepted : 8
rejected : 1
unrecognized : 0
