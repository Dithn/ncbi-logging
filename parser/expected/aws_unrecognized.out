{"accepted":true,"accession":"","agent":"aws-sdk-java/1.11.160 Linux/4.4.0-1107-aws OpenJDK_64-Bit_Server_VM/11.0.7+10/11.0.7","auth_type":"AuthHeader","bucket":"sra-pub-src-1","cipher_suite":"SigV4 ECDHE-RSA-AES128-GCM-SHA256","error":"AccessDenied","extension":"","host_header":"sra-pub-src-1.s3.amazonaws.com","host_id":"uYoz2qqYhiKUJCM3jMzDLO7the4EIjFZp6G9VLtn/jY1crCHDKHf5G3r1Cb9rmqsuOwcRrNv28k=","ip":"10.11.149.94","key":"","method":"HEAD","obj_size":"","operation":"REST.HEAD.BUCKET","owner":"7dd4dcfe9b004fb7433c61af3e87972f2e9477fa7f0760a02827f771b41b3455","path":"/","referer":"-","request_id":"186D7BF85015E3CE","requester":"arn:aws:iam::959185930736:user/david.yadin@kymab.com","res_code":"403","res_len":"243","source":"S3","time":"[26/May/2020:13:32:00 +0000]","tls_version":"TLSv1.2","total_time":"196","turnaround_time":"","vers":"HTTP/1.1","version_id":""}
{"accepted":true,"accession":"","agent":"aws-cli/1.16.300 Python/2.7.16 Linux/4.14.177-139.254.amzn2.x86_64 botocore/1.13.36","auth_type":"AuthHeader","bucket":"sra-pub-src-1","cipher_suite":"SigV4 ECDHE-RSA-AES128-GCM-SHA256","error":"NoSuchKey","extension":"","host_header":"sra-pub-src-1.s3.amazonaws.com","host_id":"9cLyZqXg5N2mXetfQMtvMhhbczeIiHy+9UxgBSDV6UkpNCZxnUN7cNN/4TDc2oBlMk1DAUtG0GA=","ip":"3.235.179.196","key":"SRR9154112/*.fastq*","method":"HEAD","obj_size":"","operation":"REST.HEAD.OBJECT","owner":"7dd4dcfe9b004fb7433c61af3e87972f2e9477fa7f0760a02827f771b41b3455","path":"/SRR9154112/%2A.fastq%2A","referer":"-","request_id":"1B6E92A991321AE8","requester":"arn:aws:sts::784757538848:assumed-role/TulipRWAccess/i-00a5d1f5e0d3cdc67","res_code":"404","res_len":"290","source":"S3","time":"[26/May/2020:16:00:16 +0000]","tls_version":"TLSv1.2","total_time":"43","turnaround_time":"","vers":"HTTP/1.1","version_id":""}
{"accepted":true,"accession":"","agent":"","auth_type":"AuthHeader","bucket":"sra-pub-src-1","cipher_suite":"SigV4 ECDHE-RSA-AES128-GCM-SHA256","error":"","extension":"","host_header":"sra-pub-sars-cov2.s3.amazonaws.com","host_id":"QS2B/AbVsYy0J+Ce10atUPXDMX6vu6UOwEYF+EEovqNazgm8aO1xGpNN29+hN0KjCYv0VOTvsdI=","ip":"130.14.28.7","key":"SRR9211507/100100.mapped.1.fq.1","method":"","obj_size":"351400","operation":"REST.COPY.OBJECT_GET","owner":"7dd4dcfe9b004fb7433c61af3e87972f2e9477fa7f0760a02827f771b41b3455","path":"","referer":"","request_id":"9713E7798F9A44BA","requester":"arn:aws:iam::783971887864:user/ignatovi","res_code":"","res_len":"","source":"S3","time":"[26/May/2020:18:42:08 +0000]","tls_version":"TLSv1.2","total_time":"","turnaround_time":"","vers":"","version_id":""}
{"accepted":true,"accession":"","agent":"aws-cli/1.16.249 Python/2.7.16 Linux/4.14.138-89.102.amzn1.x86_64 botocore/1.12.239","auth_type":"AuthHeader","bucket":"sra-pub-src-1","cipher_suite":"SigV4 ECDHE-RSA-AES128-GCM-SHA256","error":"","extension":"","host_header":"sra-pub-src-1.s3.amazonaws.com","host_id":"+IfPMMJukaeGXc7cHIdkcLUDPcCCQVPtmWh04I9UUahItPSQxFgDgNUQQbh/ysxI4n4gHf5NK7M=","ip":"52.54.203.43","key":"SRR9158220/A118_PF_4_1.fq.1","method":"GET","obj_size":"10679416372","operation":"REST.GET.OBJECT","owner":"7dd4dcfe9b004fb7433c61af3e87972f2e9477fa7f0760a02827f771b41b3455","path":"/SRR9158220/A118_PF_4_1.fq.1","referer":"-","request_id":"B20C3FBA17A4BC38","requester":"arn:aws:sts::783971887864:assumed-role/sra-developer-instance-profile-role/i-04b64e15519efb678","res_code":"206","res_len":"8388608","source":"S3","time":"[25/May/2020:22:32:08 +0000]","tls_version":"TLSv1.2","total_time":"230","turnaround_time":"182","vers":"HTTP/1.1","version_id":""}
{"accepted":true,"accession":"","agent":"aws-cli/1.16.129 Python/3.6.7 Linux/4.15.0-1032-aws botocore/1.12.119","auth_type":"AuthHeader","bucket":"sra-pub-run-1","cipher_suite":"SigV4 ECDHE-RSA-AES128-GCM-SHA256","error":"NoSuchKey","extension":"","host_header":"sra-pub-run-1.s3.amazonaws.com","host_id":"dPgBpCGq9rQ7ZciElvjyXYPzKZV/WO1Kzthf39aEE7/VJiuOve05tKd5zgaCJlWYXnS89nw+YFo=","ip":"3.89.6.210","key":".","method":"HEAD","obj_size":"","operation":"REST.HEAD.OBJECT","owner":"922194806485875312b252374a3644f1feecd16802a50d4729885c1d11e1fd37","path":"/.","referer":"-","request_id":"31D289F437B52130","requester":"arn:aws:iam::783971887864:user/ignatovi","res_code":"404","res_len":"272","source":"S3","time":"[22/Mar/2019:18:05:28 +0000]","tls_version":"TLSv1.2","total_time":"23","turnaround_time":"","vers":"HTTP/1.1","version_id":""}
{"accepted":true,"accession":"","agent":"aws-cli/1.16.129 Python/3.6.7 Linux/4.15.0-1032-aws botocore/1.12.119","auth_type":"AuthHeader","bucket":"sra-pub-run-1","cipher_suite":"SigV4 ECDHE-RSA-AES128-GCM-SHA256","error":"NoSuchKey","extension":"","host_header":"sra-pub-run-1.s3.amazonaws.com","host_id":"dPgBpCGq9rQ7ZciElvjyXYPzKZV/WO1Kzthf39aEE7/VJiuOve05tKd5zgaCJlWYXnS89nw+YFo=","ip":"3.89.6.210","key":".","method":"HEAD","obj_size":"","operation":"REST.HEAD.OBJECT","owner":"922194806485875312b252374a3644f1feecd16802a50d4729885c1d11e1fd37","path":"/.","referer":"-","request_id":"31D289F437B52130","requester":"arn:aws:iam::783971887864:user/ignatovi","res_code":"404","res_len":"272","source":"S3","time":"[22/Mar/2019:18:05:28 +0000]","tls_version":"TLSv1.2","total_time":"23","turnaround_time":"","vers":"HTTP/1.1","version_id":""}
